tests/gh-actions/install_KinD_create_KinD_cluster_install_kustomize.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+# Function to handle errors
+handle_error() {
+    echo "Error: $1"
+    exit 1
+}
+
+# Install KinD
+echo "Installing KinD..."
+if ! ./tests/gh-actions/install_kind.sh; then
+    handle_error "Failed to install KinD"
+fi
+
+# Create KinD Cluster
+echo "Creating KinD Cluster..."
+if ! kind create cluster --config tests/gh-actions/kind-cluster.yaml; then
```

Comment:
Can we inline all three scripts to save 3 additional files?

Reply:
I have addressed this. Can you check it? Let me know if you have any concerns. 
If not I can delete those redundant files and update the PR